README.md

```diff
@@ -258,8 +258,10 @@ Web UI Options
 
 Setup command
 
-  -o, --otc string   Setup device using a one time code
-  -u, --ff-url url   URL of FlowFuse. Required for setup
+  -o, --otc string         Setup device using a one time code
+  -s, --otc-dont-start     Do not start the agent after setup
+  -n, --otc-dont-import    Do not ask to import Node-RED flows during setup
```

Comment:
I had proposed in the FF pr to change these flags - without realising only one of them is 'new' in this PR, which makes it a bit trickier to propose changes here.

We haven't released the `-s` flag yet, so we can still change if needed.

My concern is the short versions `-s` and `-n` feel a little random - would we be better to not have short-codes and just have the long form for these two?

Reply:
sensible. Will address.